kata-deploy/kata-cleanup.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kubelet-cri-o-kata-cleanup
+  namespace: kube-system
+spec:
+  selector:
+      matchLabels:
+        name: kubelet-cri-o-kata-cleanup
+  template:
+    metadata:
+        labels:
+          name: kubelet-cri-o-kata-cleanup
+    spec:
+      serviceAccountName: kata-label-node
+      nodeSelector:
+          container-runtime: cri-o
+          kata-runtime: cleanup
+      containers:
```

Comment:
need volume mount for systemd and dbus just like kata-deploy.yaml below?

Reply:
Yes, agreed.